src/mechanics/cell/DifferentialOperators.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#include <Eigen/Core>
+#include "base/Exception.h"
+
+namespace NuTo
+{
+
+namespace Nabla
```

Comment:
IMO, nabla is a specific operator, this naming of the namespace might lead to confusion

Reply:
Added to #124.